### ISSUE TITLE
Map card: "zoom" buttons may be shown without "+/-" labels (and colors should be consistent)

### ISSUE THREAD

**Issue by ildar170975** [home-assistant/frontend#18990]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Consider these cards:
```

  - type: horizontal-stack
    cards:
      - type: map
        entities:
          - person.ildar
      - type: map
        entities:
          - person.ildar
        dark_mode: false
      - type: map
        entities:
          - person.ildar
        dark_mode: true
```
How the "zoom" buttons are shown:

light theme (not visible if `dark_mode: true`):

![изображение](https://github.com/home-assistant/frontend/assets/71872483/1b003f3b-a9c1-4280-beb3-ef91ed6216a4)

dark theme:

![изображение](https://github.com/home-assistant/frontend/assets/71872483/97c029d8-3abc-424a-aa79-3c87eaffa307)

Imho a button color should follow a theme independently on the `dark_mode` value.

### Describe the behavior you expected

see above

### Steps to reproduce the issue

as above

### What version of Home Assistant Core has the issue?

2023.12.1

### In which browser are you experiencing the issue with?

Chrome 119.0.6045.200 

### Which operating system are you using to run this browser?

Win10x64

**Comment by ildar170975** [comment]
2024.3
still there:
![image](https://github.com/home-assistant/frontend/assets/71872483/f9519150-126e-41c0-a1cc-6d5e083ed14b)

**Comment by ildar170975** [comment]
fixed in 2024.5
